{"uuid":"fb08b07d-0477-47fd-811f-0b016eda1a06","name":"Default test","children":["b308d70d-21d9-4795-9a46-c02eb257a027","d3a86c33-1e34-4efa-bddb-2bd3d624b720","5f504dac-0f96-496f-b1c6-9efbb266874e","09c9c845-8672-47df-9aa1-20ca8b21eb74","38319d88-bdb3-431c-9dec-f199a6759647","81ee1afc-dc48-430a-bf70-a6911460bba7","08da645f-8129-46a8-b788-6dd5ee5ba76e","723ffd64-7d46-469d-90a7-392ec72ab0c8"],"befores":[{"name":"setUp","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"chromedriver located at path/to/chromedriver.exe, but invalid\nFor documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/\nBuild info: version: '4.12.1', revision: '8e34639b11'\nSystem info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.8.1'\nDriver info: driver.version: ChromeDriver","trace":"org.openqa.selenium.remote.NoSuchDriverException: chromedriver located at path/to/chromedriver.exe, but invalid\nFor documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/\nBuild info: version: '4.12.1', revision: '8e34639b11'\nSystem info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.8.1'\nDriver info: driver.version: ChromeDriver\r\n\tat org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:45)\r\n\tat org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:13)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.generateExecutor(ChromeDriver.java:99)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:88)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:83)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:51)\r\n\tat runner.FeatureRunner.setUp(FeatureRunner.java:21)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)\r\n\tat org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:663)\r\n\tat org.testng.TestRunner.beforeRun(TestRunner.java:652)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\r\n\tat org.testng.TestNG.run(TestNG.java:1082)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1699622530041,"stop":1699622530157}],"afters":[{"name":"tearDown","status":"broken","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1699622530852,"stop":1699622530852}],"start":1699622529997,"stop":1699622530853}